src/js/types.ts

```diff
@@ -310,8 +317,8 @@ export interface API {
   lockFilePromise: Promise<any>;
   lockfile_unvendored_stdlibs: string[];
   lockfile_unvendored_stdlibs_and_test: string[];
-  lockfile_info: any;
-  lockfile_packages: any;
+  lockfile_info: LockfileInfo;
+  lockfile_packages: Record<string, PackageData>;
   repodata_packages: any;
   repodata_info: any;
```

Comment:
`repodata_packages` and `repodata_info` are synonyms for `lockfile_packages` and `lockfile_info` so could you give them the same type?